src/main/java/com/violeth/blockjs/blockjs/jsinterface/mcinterface/Player.java

```diff
@@ -0,0 +1,152 @@
+package com.violeth.blockjs.blockjs.jsinterface.mcinterface;
+
+import com.eclipsesource.v8.V8;
+import com.eclipsesource.v8.V8Array;
+import com.eclipsesource.v8.V8Function;
+import com.violeth.blockjs.blockjs.BlockJS;
+import org.bukkit.Bukkit;
+import org.bukkit.GameMode;
+
+import java.util.Collection;
+import java.util.UUID;
+
+public class Player {
+    V8 v8;
+    public Player(V8 v8) {
+        this.v8 = v8;
+    }
+    public void setGameMode(String playerUUID, GameMode gameMode) {
```

Comment:
`setGameMode()` can't take GameMode as an argument, as there is no GameMode enum in JavaScript. Maybe change this to an int and add a case for mapping the int to the enum, or something along those lines?